OurUmbraco.Site/web.vsts.config

```diff
@@ -579,6 +579,28 @@
                     <match url="rss/wiki*" />
                     <action type="Redirect" url="/Documentation/" redirectType="Permanent" appendQueryString="true" />
                 </rule>
+		<!-- START - Renaming Getting Started to Fundamentals -->
```

Comment:
I would not use the "START" and "END" comments as someone might add a new line in between, otherwise, I can not see a mistake